conf/sources.json

```diff
@@ -3,13 +3,15 @@
     "prefix_cluster1_stream_alert_kinesis": {
       "logs": [
         "cloudwatch",
-        "ghe"
+        "ghe",
```

Comment:
why is this file changing?

Reply:
she added test rule tests that are osquery logs

Reply:
^^ What's this guy talking about? no she didn't.. :)

Reply:
I will remove this changing 😄